web/src/components/login-page/index.tsx

```diff
@@ -101,7 +101,7 @@ export const LoginPage: FC = memo(function LoginPage() {
               />
               {isPlayEnvironment && (
                 <div className={classes.note}>
-                  Try with <strong>play</strong> name if you want to join the
+                  Try project name <strong>play</strong> if you want to join the
```

Comment:
nits
```suggestion
                  Try to input <strong>play</strong> as project name if you want to join the
```

Reply:
Nice idea, but this text is a bit too long to be shown in a single line 🙈

Reply:
Oh, I got it.
Does it not also work if you set `testAlign: "left"`?

Reply:
align right
<img width="675" alt="Screen Shot 2022-03-23 at 9 32 10" src="https://user-images.githubusercontent.com/32532742/159612416-1267e443-98f3-4772-8932-27aac0512092.png">

align left
<img width="697" alt="Screen Shot 2022-03-23 at 9 32 53" src="https://user-images.githubusercontent.com/32532742/159612515-c16b5abd-cb1c-4664-83db-ee22c482bfb4.png">

🙈

Reply:
😇 
allright, let's keep it that way.